features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionView.kt

```diff
@@ -121,58 +110,22 @@ internal fun HeaderContent(verificationFlowStep: FlowStep, modifier: Modifier =
         FlowStep.Ready, is FlowStep.Verifying, FlowStep.Completed -> R.string.screen_session_verification_compare_emojis_subtitle
     }
 
-    Column(modifier) {
-        Spacer(Modifier.height(80.dp))
-        Box(
-            modifier = Modifier
-                .size(width = 70.dp, height = 70.dp)
-                .align(Alignment.CenterHorizontally)
-                .background(
-                    color = LocalColors.current.quinary,
-                    shape = RoundedCornerShape(14.dp)
-                )
-        ) {
-            Spacer(modifier = Modifier.height(68.dp))
```

Comment:
This line has been removed, since I found it useless. Did I miss something?

Reply:
I can't remember, to be honest. If you test it in a small display (Nexus 5?) and it works fine, then it's probably not needed anymore.